v1/edgeloadbalancer/policies_http_security_types.go

```diff
@@ -0,0 +1,159 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025 Orange
+ * SPDX-License-Identifier: Mozilla Public License 2.0
+ *
+ * This software is distributed under the MPL-2.0 license.
+ * the text of which is available at https://www.mozilla.org/en-US/MPL/2.0/
+ * or see the "LICENSE" file for more details.
+ */
+
+package edgeloadbalancer
+
+import (
+	govcdtypes "github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+type (
+	PoliciesHTTPSecurityModel struct {
+		VirtualServiceID string `validate:"required,urn=loadBalancerVirtualService"`
+		// List of HTTP Request Policies
+		Policies []*PoliciesHTTPSecurityModelPolicy `validate:"required,dive"`
+	}
+
+	PoliciesHTTPSecurityModelPolicy struct {
+		// Name of the rule
+		Name string `validate:"required"`
+		// Whether the rule is active or not.
+		Active bool `validate:"omitempty"`
+		// Whether to enable logging for the rule (policy)
+		Logging bool `validate:"omitempty"`
+		// MatchCriteria for the HTTP Request
+		MatchCriteria PoliciesHTTPSecurityMatchCriteria `validate:"required"`
+
+		// Action to take when the rule matches
+
+		// HTTP Connection Action
+		// It can be configured in combination with other actions
+		ConnectionAction string `validate:"omitempty,oneof=ALLOW CLOSE"`
+		// HTTP Rate Limit Action
+		// It can be configured in combination with other actions
+		RateLimitAction *PoliciesHTTPActionRateLimit `validate:"omitempty"`
+		// HTTP Redirect to HTTPS Action
+		// It can be configured in combination with other actions
+		RedirectToHTTPSAction *int `validate:"omitempty"`
+		// HTTP Send Response Action
+		// It can be configured in combination with other actions
+		SendResponseAction *PoliciesHTTPActionSendResponse `validate:"omitempty"`
+	}
+
+	PoliciesHTTPSecurityMatchCriteria struct {
+		// Protocol
+		Protocol string `validate:"omitempty,oneof=HTTP HTTPS"`
```

Comment:
```suggestion
		// Protocol
		Protocol PoliciesHTTPProtocol `validate:"omitempty,oneof=HTTP HTTPS"`
```

Add ref. const PoliciesHTTPProtocols